### PR TITLE
Ansyn ol

### PR DESCRIPTION
**Pull request by yairtawil**

### CODE REVIEW

File: confd/templates/production.tmpl [hunk 1]
```diff
@@ -320,7 +320,7 @@
     "storageService": {
       "baseUrl": "{{ getv "/ansyn/storage/baseurl" "http://platform.ansyn.io:8081/api/store" }}"
     },
-    "translation": {
+    "translation": {src/app/@ansyn/ansyn/app-providers/overlay-source-providers/idaho-source-provider.ts
```
Comment (asafMasa) [inline confd/templates/production.tmpl]: ?

File: src/app/@ansyn/map-facade/components/imagery-rotation/imagery-rotation.component.ts [hunk 2]
```diff
@@ -73,6 +71,10 @@ export class ImageryRotationComponent {
 				protected store: Store<any>) {
 	}
 
+	toDegrees(radians: number): number {
```
Comment (asafMasa) [inline src/app/@ansyn/map-facade/components/imagery-rotation/imagery-rotation.component.ts]: you can move this method to "imagery
ol and mapFacade knows it

File: src/app/@ansyn/ol/mapSourceProviders/marco/open-layers-MARCO-source-provider.ts [hunk 3]
```diff
@@ -149,27 +124,44 @@ export class OpenLayerMarcoSourceProvider extends OpenLayersMapSourceProvider<IM
...
+	// getThumbnailUrl(overlay: IOverlay) {
+	// 	let { imageUrl, thumbnailUrl } = overlay;
+	//
+	// 	if (thumbnailUrl) {
+	// 		return of(thumbnailUrl);
+	// 	}
+	//
+	// 	imageUrl = this.encodeImagePath(imageUrl);
+	//
+	// 	return this.getWMSCapabilities(imageUrl).pipe(
```
Comment (asafMasa) [inline src/app/@ansyn/ol/mapSourceProviders/marco/open-layers-MARCO-source-provider.ts]: all marked? is it working

File: confd/templates/production.tmpl (hunk 1 above)
Comment (yairtawil) [reply confd/templates/production.tmpl]: 👍

File: src/app/@ansyn/map-facade/components/imagery-rotation/imagery-rotation.component.ts (hunk 2 above)
Comment (yairtawil) [reply src/app/@ansyn/map-facade/components/imagery-rotation/imagery-rotation.component.ts]: just on line..

File: src/app/@ansyn/ol/mapSourceProviders/marco/open-layers-MARCO-source-provider.ts (hunk 3 above)
Comment (yairtawil) [reply src/app/@ansyn/ol/mapSourceProviders/marco/open-layers-MARCO-source-provider.ts]: move to overlay source provider...

**Pull request merged**